README.md

```diff
@@ -42,21 +42,24 @@ var stats = readingTime(text);
 ```javascript
 var readingTime = require('reading-time/stream');
 
-fs.createReadStream('foo').pipe(readingTime).on('data', function(stats) {
-	// ...
-});
+fs.createReadStream('foo')
+  .pipe(readingTime)
+  .on('data', function(stats) {
+    // ...
+  });
 ```
 
 ## API
 
-`readingTime(text, options)`
+`readingTime(text, options?)`
 
- - `text`: the text to analyze
- - `options.wordsPerMinute`: the words per minute an average reader can read (default: 200)
- - `options.wordBound`: a function than return if a character is considered as a word bound (default: spaces, new lines and tabulations)
+- `text`: the text to analyze
+- options (optional)
+- `options.wordsPerMinute`: (optional) the words per minute an average reader can read (default: 200)
+- `options.wordBound`: (optional) a function that returns a boolean value depending on if a character is considered as a word bound (default: spaces, new lines and tabulations)
```

Comment:
I'm fine with adding the `options` explicitly here, but I would slightly change the formatting then:
```md
- `options`: (optional)
  - `wordsPerMinute`: (optional) the words per minute an average reader can read (default: 200)
  - `wordBound`: (optional) a function that returns a boolean value depending on if a character is considered as a word bound (default: spaces, new lines and tabulations)
```